tests/org.eclipse.n4js.spec.tests/xpect-tests/Ch05_04_01_Overriding_Members/Req72_Overriding_Members/accessors/Req72_err_PartiallyOverriding_FieldAccessors_in_JS.js.xt

```diff
@@ -15,7 +15,8 @@ class A {
 	set a(v) {}
 }
 class B extends A {
-	/* XPECT warnings ---
+	// disabled issues until GH-855 is solved
+	/* X-PECT warnings ---
```

Comment:
it will be easy to overlook all those tests. Add note in #855 that this pattern was used to disable tests.